src/gs-overview-page.c

```diff
@@ -35,6 +35,7 @@
 #include "gs-common.h"
 
 #define N_TILES 35
+#define MIN_POPULAR_TILES 2
```

Comment:
I'm not sure why we bother setting a minimum of 2 -- if someone sets up an image to have just one popular title, it seems like we could respect it.  In practice, I doubt we will ever set just one, so I'm going to accept as is, but in the future we might want to reduce the minimum to 1.